code/modules/mob/living/living_defense.dm

```diff
@@ -191,6 +191,8 @@
 	mind.transfer_to(target)
 	target.mental_dominator = src
 	spawn(duration)

```

Comment:
to the scripture, so there's a message if the invoker manages to successfully return